vcx/ci/Jenkinsfile

```diff
@@ -97,9 +97,6 @@ def mainUbuntu() {
                 echo "$ex error"
             } finally {
 				sh 'docker system df'
-                sh "docker rm ${containerNames[0]}"
```

Comment:
I think we should keep this but just add functionality to check if the container exists. Fellipe mentioned something like "docker rm -f ${containerNames[0]} || true"

Reply:
Okay, but lets not let adding that new feature block this PR.